LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Eddie Jaoude
+Copyright (c) 2021 - present Eddie Jaoude
```

Comment:
```suggestion
Copyright (c) 2021 - 2022 Eddie Jaoude
```
Or in my understanding also correct could be
```suggestion
Copyright (c) 2021, 2022 Eddie Jaoude
```

Reply:
Maybe this too?

```suggestion
Copyright (c) 2021-2022 Eddie Jaoude
```